lib/index.js

```diff
@@ -92,12 +94,54 @@ bedrock.events.on('bedrock-express.configure.routes', app => {
         });
     }));
 
+  app.get(
+    adminPath,
+    ensureAuthenticated,
+    intParser(),
+    validate({query: 'account.pagination'}),
+    asyncHandler(async (req, res) => {
+      const {email, cursor = null, limit = 10} = req.query;
+      const {actor} = req.user || {};
+      const query = {"account.email": email};
+      if(cursor) {
+        query["account.created"] = {$gte: cursor};
+      }
+      const options = {limit, sort: {"account.created": -1}};
+      const records = await brAccount
+        .getAll({actor, query, options});
+      res.status(200).json(records);
+    }));
+
   app.patch(
     accountPath,
     ensureAuthenticated,
-    asyncHandler(async (req, res, next) => {
-      // TODO: call `brAccount.update`
-      next();
+    intParser(),
+    validate('account.update'),
+    asyncHandler(async (req, res) => {
+      /**
+       * @function accountUpdateHandler
+       * @param {Request} req
+       * @param {Response} res
+       * @param {Function} next
+       * @description patch requests to /:account result in an updated account
+       * patches need to be in the:
+       * [json patch]{@link https://www.npmjs.com/package/fast-json-patch}
+       * format. The body needs to contains the id, patch, and sequence keys
+       */
+      try {
+        const {actor} = req.user || {};
+        const {account: id} = req.params;
+        const updater = {
+          actor,
+          id,
+          ...req.body
+        };
+        await brAccount.update(updater);
+        res.status(200).send();
```

Comment:
consider an appropriate status code instead of 200, maybe 204, whatever is appropriate for patch operations.